components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -238,22 +239,39 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         response.sendRedirect(getRedirectURL(redirectURL, request));
     }
 
+    private String getOPBSCookieValue(HttpServletRequest request) {
```

Comment:
Need debug logs to indicate how the obps value was resolved.

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1519/commits/dda7bf80cc7f604b87669ae653ed1ac12089cbb2.